packages/react-sdk/package.json

```diff
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "@nanostores/react": "^0.4.1",
+    "@radix-ui/react-tabs": "^1.0.0",
```

Comment:
Maybe we should have something like radix-sdk for such stuff?

Reply:
Yes, I thought about this too, even more so I would basically want to let user search and add all available component packages aka marketplace ... just not sure when to start with this, since we need to think the UX through for how this will work, including versioning.

Additionally we need to change how we generate props jsons from types to support external webstudio components, they should come directly prebuilt with everything, ready to get installed